rfcs/0093-rfc-categories.md

```diff
@@ -0,0 +1,152 @@
+---
+feature: rfc_categories
+start-date: 2021-05-19
+author: David Arnold
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @kevnicox, @Mic92, @gytis-ivaskevicius
+shepherd-leader: @kevincox
+related-issues:
+category: process
+---
+
+# Summary
+[summary]: #summary
+
+The RFC process is amended with means to categorize RFCs into one of: _feature_, 
+_information_ or _process_, where each category sets different accents that
+improve the overall outcome of the process.
```

Comment:
```suggestion
_information_ or _process_, where each category sets different expectations for reviewers that
improve the overall outcome of the process.
```

I'm not familiar with the use of "accents" in that context.

Additionally I feel that adding "for reviewers" emphasizes that this isn't intended to change the meaning of the final outcome or change things for authors but to help guide the review.